src/SqlAsyncCollector.cs

```diff
@@ -237,22 +252,57 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
         /// to upsert to and returns the extra property names in a List.
         /// </summary>
         /// <param name="columns"> The columns of the table to upsert to </param>
+        /// <param name="rows"> The rows to be upserted </param>
```

Comment:
It would be good to have this actually describe how the rows are used in this function - it doesn't really have anything to do with upserting.